cmd/admin-inspect.go

```diff
@@ -97,7 +106,15 @@ func mainAdminInspect(ctx *cli.Context) error {
 	splits := splitStr(aliasedURL, "/", 3)
 	bucket, prefix := splits[1], splits[2]
 
-	key, r, ierr := client.Inspect(context.Background(), madmin.InspectOptions{Volume: bucket, File: prefix})
+	shellName, _ := getShellNave()
+	if shellName != "bash" && strings.Contains(prefix, "*") {
+		console.Infoln("Your shell is auto determined as '" + shellName + "', wildcard patterns are only supported with 'bash' SHELL.")
```

Comment:
/cc @klauspost

Reply:
It works fine on Windows. @harshavardhana Why is this being diabled?

Reply:
on zsh and tcsh shells `**` is escaped by the shell and it doesn't work.

Reply:
Then exclude those explicitly or have an OS check.

Reply:
fixed @klauspost